ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualSignedTransaction.kt

```diff
@@ -25,7 +24,10 @@ import java.security.PublicKey
  * Thus adding or removing a signature does not change it.
  */
 @DoNotImplement
-@CordaSerializable
+// Why do we make it illegal to have a custom serializer for a corda serializable annotated class/interface?
+// This seems very dangerous moving forward if we need to provide custom serializers for existing code
+// net.corda.internal.serialization.amqp.CachingCustomSerializerRegistry - Illegal custom serializer detected for interface net.corda.v5.ledger.consensual.transaction.ConsensualSignedTransaction: net.corda.internal.serialization.amqp.standard.CustomSerializer.Proxy
+//@CordaSerializable
```

Comment:
Is there a follow up ticket/thread for this? Clearly not an essential blocker on this PR, but needs to be sorted out in general.

Reply:
https://r3-cev.atlassian.net/browse/CORE-6971 was created a few days ago.